snapcraft/internal/pluginhandler/_patchelf.py

```diff
@@ -49,7 +49,7 @@ def __init__(
         :param confinement: the confinement value the snapcraft project is
                             using (i.e.; devmode, strict, classic).
         :param core_base: the base the snap will use during runtime
-                          (e.g.; core, core18).
+                          (e.g.; core, core18, core20).
```

Comment:
this change is not really necessary, it is an e.g. after all.

Reply:
reverted